src/cr/cube/dimension.py

```diff
@@ -896,11 +906,14 @@ class Element:
     This object resolves the transform cascade for element-level transforms.
     """
 
-    def __init__(self, element_dict, index, element_transforms, label_formatter):
+    def __init__(
+        self, element_dict, index, element_transforms, label_formatter, dim_type
```

Comment:
Maybe `dim_type` can be an optional value with default value `None`.

Reply:
well this should be mandatory, so I do prefer to change the unit tests with `None` in order to prevent any future regression